src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -155,13 +155,13 @@ class MulticopterPositionControl : public ModuleBase<MulticopterPositionControl>
 		(ParamFloat<px4::params::MPC_Z_VEL_MAX_DN>) _vel_max_down,
 		(ParamFloat<px4::params::MPC_LAND_SPEED>) _land_speed,
 		(ParamFloat<px4::params::MPC_TKO_SPEED>) _tko_speed,
-		(ParamFloat<px4::params::MPC_LAND_ALT2>) MPC_LAND_ALT2, /**< downwards speed limited below this altitude */
-		(ParamInt<px4::params::MPC_POS_MODE>) MPC_POS_MODE,
-		(ParamInt<px4::params::MPC_AUTO_MODE>) MPC_AUTO_MODE,
-		(ParamInt<px4::params::MPC_ALT_MODE>) MPC_ALT_MODE,
-		(ParamFloat<px4::params::MPC_SPOOLUP_TIME>) MPC_SPOOLUP_TIME, /**< time to let motors spool up after arming */
-		(ParamInt<px4::params::MPC_OBS_AVOID>) MPC_OBS_AVOID, /**< enable obstacle avoidance */
-		(ParamFloat<px4::params::MPC_TILTMAX_LND>) MPC_TILTMAX_LND /**< maximum tilt for landing and smooth takeoff */
+		(ParamFloat<px4::params::MPC_LAND_ALT2>) _land_alt2, /**< downwards speed limited below this altitude */
+		(ParamInt<px4::params::MPC_POS_MODE>) _pos_mode,
+		(ParamInt<px4::params::MPC_AUTO_MODE>) _auto_mode,
+		(ParamInt<px4::params::MPC_ALT_MODE>) _alt_mode,
+		(ParamFloat<px4::params::MPC_SPOOLUP_TIME>) _spoolup_time, /**< time to let motors spool up after arming */
+		(ParamInt<px4::params::MPC_OBS_AVOID>) _obs_avoid, /**< enable obstacle avoidance */
+		(ParamFloat<px4::params::MPC_TILTMAX_LND>) _tiltmax_lnd /**< maximum tilt for landing and smooth takeoff */
```

Comment:
One last thought that might benefit readers in the future.... if you use the entire PARAM_NAME, it might add a better trail of breadcrumbs.  e.g. `_mpc_tiltmax_lnd`

I realize that some of the `_class_member_vars` already in place are not doing that, but the example in the developer shows this usage and it certainly leaves no ambiguity as to which param it is referring to.

Reply:
I agree that having the full name is better, I have changed the PR wo WIP, so I will get some more responses and then do new clean PR

Reply:
@BazookaJoe1900 , you can get a clean result in this PR by rebasing: https://dev.px4.io/en/contribute/git_examples.html.  It's a straightforward process once you've been through it the first time.  Feel free to find me on PX4 slack if I can help with questions on rebasing.